src/components/area.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import clsx from 'clsx'
+
+import { TextareaBase } from '../base-components'
+
+
+const _areaClasses = ({ container }) => ({
+  container: `${container ? container : 'w-96 flex flex-col'}`,
+  label: 'text-secondary-600',
+  helperText: 'mt-1.5 text-secondary-600',
+  wordCount: 'mt-1.5 col-start-2 justify-self-end text-secondary-600 text-xxs tracking-lg leading-1.6',
+})
+
+const _textareaBaseClasses = ({ focus, root, filled, disabled }) => ({
+  root: clsx(`${root ? root : 'w-96 h-24 mt-1.5 rounded-sm p-sm font-sans text-sm tracking-sm leading-1.43'}`,
+    { 'border-secondary-600': !disabled },
+    { 'border-interactive-500 shadow-focused-interactive': focus },
+    { 'border-interactive-500 bg-neutral-50': filled },
```

Comment:
I think in our guide we didn't have the filled part.  Please check the Zeplin or the design guide again. Because `bg-neutral-50` is a little bit blue. And give people a feeling they are still editing it.

If you want to keep this filled, please change it to `secondary-50`
![image](https://user-images.githubusercontent.com/20359409/126835295-ea35bbbe-db59-4df4-9b4d-1bffefaaccd0.png)

Reply:
will do secondary-50 then